opensrp-anc/build.gradle

```diff
@@ -153,9 +152,7 @@ tasks.withType(Test) {
 
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.3.1'
-    //implementation(project(':android-json-form-wizard'))
-    implementation('org.smartregister:opensrp-client-native-form:2.1.15-SNAPSHOT@aar')
-    {
+    implementation('org.smartregister:opensrp-client-native-form:2.1.15-dev-3-SNAPSHOT@aar') {
```

Comment:
Please update this to an actual tag that's not `dev`

Reply:
Done